modules/light-clients/xx-mock/types/client_state.go

```diff
@@ -113,7 +115,35 @@ func (cs ClientState) VerifyMembership(
 		return sdkerrors.Wrap(clienttypes.ErrConsensusStateNotFound, "please ensure the proof was constructed against a height that exists on the client")
 	}
 
-	h := sha256.Sum256(value)
+	// sha256(abi.encodePacked(height.toUint128(), sha256(prefix), sha256(path), sha256(value)))
+	revisionNumber := height.GetRevisionNumber()
+	revisionHeight := height.GetRevisionHeight()
+
+	heightBuf := make([]byte, 16)
+	binary.LittleEndian.PutUint64(heightBuf[:8], revisionHeight)
+	binary.LittleEndian.PutUint64(heightBuf[8:], revisionNumber)
```

Comment:
Please use `BigEndian` and put `revisionNumber` at first and `revisionHeight` at last.

Reply:
Fixed
https://github.com/datachainlab/ibc-mock-client/pull/11/commits/331c660cb57dbeb822ab4a6ff80e33b7953c9fc4